docs/2.8.0/docs/canton/usermanual/installation.rst

```diff
@@ -120,14 +120,21 @@ Persistence using Postgres
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 While in-memory is great for testing and demos, for more serious tasks, you need to use a database as a persistence layer.
-Both the community version and the enterprise version support Postgres as a persistence layer. Make sure that you have
-a running Postgres server and you need to create one database per node. The recommended Postgres version to use is 11,
-as this is tested the most thoroughly.
+Both the community version and the enterprise version support `Postgres <https://www.postgresql.org/>`__ as a persistence
+layer. Make sure that you have a running Postgres server and create one database per node.
+
+Generally speaking, Canton is tested on
+`currently supported Postgres versions <https://www.postgresql.org/support/versioning/>`__ only.
+Please refer to the `Canton's release notes <https://github.com/digital-asset/canton/releases>`__ for the specific
+Postgres version which has been used to test a particular Canton release.
```

Comment:
```suggestion
Both the community version and the enterprise version support `Postgres <https://www.postgresql.org/>`__ as a persistence
layer. Make sure you have a running Postgres server and create one database per node.

Canton is tested on
`currently supported Postgres versions <https://www.postgresql.org/support/versioning/>`__ only.
See the `Canton release notes <https://github.com/digital-asset/canton/releases>`__ for the specific
Postgres version used to test a particular Canton release.
```

Reply:
I intentionally used `Generally speaking,` to soften the statement because Canton is not necessarily tested on all Postgres DB versions supported by Postgres.

For example, Canton is currently tested on Postgres 11, 12, 13, 14, 15. Postgres 11 becomes EOL (end-of-life, unsupported) on November 9th 2023 (tomorrow). The latest currently supported Postgres version is 16, but we don't test it yet as it has just been released recently.

Reply:
@marciw-da I changed the text, please have look.